Automattic/wp-calypso
# Adds an error notice for Jetpack Backup when the automated transfer fails

#### Changes proposed in this Pull Request

* Adds an error for when the automated transfer process fails.

#### Testing instructions



* Create a site with a Business plan.
* Generate an issue, typically by a reverted transfer. See the ticket for details.
* Go to Jetpack > Backup with the feature section enabled.
* Verify that the activate button is active.
* Click the button.
* Verify an error appears as below.

#### Screenshots
<img width="755" alt="Screen Shot 2020-06-19 at 4 24 20 PM" src="https://user-images.githubusercontent.com/1760168/85177598-75173280-b24a-11ea-8d86-6cceb5bab4ba.png">

Fixes 1179060693083348-as-1181203155439417
